components/tenant-mgt/org.wso2.carbon.tenant.mgt/src/main/java/org/wso2/carbon/tenant/mgt/services/TenantMgtAdminService.java

```diff
@@ -196,14 +196,14 @@ public String addSkeletonTenant(TenantInfoBean tenantInfoBean) throws Exception
         int tenantId;
         checkIsSuperTenantInvoking();
         try {
-            tenantId=TenantMgtServiceComponent.getTenantManager().getTenantId(tenantInfoBean.getTenantDomain());
+            tenantId = TenantMgtServiceComponent.getTenantManager().getTenantId(tenantInfoBean.getTenantDomain());
         } catch (org.wso2.carbon.user.api.UserStoreException e) {
             String msg = "Error in getting tenant id";
             log.error(msg, e);
             throw new Exception(msg, e);
         }
-        if(tenantId<0){
-            String msg = "Tenant is not added in user store "+tenantInfoBean.getTenantDomain();
+        if (tenantId < 0) {
+            String msg = "Tenant is not added in user store " + tenantInfoBean.getTenantDomain();
```

Comment:
Is this msg correct?

Reply:
I hope so. It seems the message is shown when the tenant is the super-tenant. Any suggestion?

Reply:
Fixed in #207